modules/electrophysiology_browser/php/electrophysiologybrowserrowprovisioner.class.inc

```diff
@@ -47,10 +47,17 @@ class ElectrophysiologyBrowserRowProvisioner
               s.Visit_label                             AS Visit_Label,
               MIN(pf.AcquisitionTime)                   AS Acquisition_Time,
               MIN(pf.InsertTime)                        AS Insert_Time,
-              GROUP_CONCAT(DISTINCT pot.OutputTypeName) AS Links,
+              GROUP_CONCAT(DISTINCT CASE 
+                WHEN pot.OutputTypeName='derivatives' THEN 'raw,derivative'
+                ELSE 'raw'
+              END) AS Links,
+              GROUP_CONCAT(DISTINCT CASE 
+                WHEN pot.OutputTypeName='derivatives' THEN 'Derivative'
```

Comment:
Should be WHEN pot.OutputTypeName='derivatives' THEN 'Derivatives' to be consistent with the row entries (since the recommended change above).

Reply:
If you look at Christine's requested changes above, she requested that I change all mentions of "Derivatives" to singular and not plural

Reply:
Therefore the sessions class needs an update, and ideally the naming convention between the frontend and the database: [SQL/0000-00-05-ElectrophysiologyTables.sql](https://github.com/aces/Loris/blob/main/SQL/0000-00-05-ElectrophysiologyTables.sql#L210)